### PR TITLE
use utf-8 characters for default cli formatter

### PR DESCRIPTION
**Pull request by arlimus**
see https://github.com/chef/inspec/issues/532

screenshot from 2016-06-15 19-28-40

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:

* @arlimus

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

File: lib/inspec/rspec_json_formatter.rb [hunk 1]
```diff
@@ -151,14 +151,14 @@ class InspecRspecCli < InspecRspecJson # rubocop:disable Metrics/ClassLength
...
   INDICATORS = {
-    'critical' => '[CRIT]   ',
-    'major'    => '[FAIL]   ',
-    'minor'    => '[WARN]   ',
-    'failed'   => '[FAIL]   ',
-    'skipped'  => '[SKIP]   ',
-    'passed'   => '[PASS]   ',
-    'unknown'  => '[ ?? ]   ',
-    'empty'    => '         ',
+    'critical' => '  ✖  ',
```
Comment (chris-rock) [inline lib/inspec/rspec_json_formatter.rb:154]: cool improvement

**Comment by chris-rock** [comment]
So cool to see more UTF-8 magic

**Comment by fcaviggia** [comment]
Rock on!  This is helpful.

**Comment by aaronlippold** [comment]
👍

**Pull request merged**